qrisk/tests/test_stats.py

```diff
@@ -79,49 +137,140 @@ def test_annual_volatility(self, returns, period, expected):
         )
 
     @parameterized.expand([
-        (simple_rets, 1.2333396776895436),
+        (s_2.pct_change().dropna(), -2.3992211554712197)
+    ])
+    def test_calmar(self, returns, expected):
+        self.assertAlmostEqual(
+            stats.calmar_ratio(
+                returns),
+            expected,
+            DECIMAL_PLACES)
+
+    @parameterized.expand([
+        (s_1, 0.0, 2.0)
```

Comment:
should we add some cases with a non-zerio minimum threshold, this is just positive returns over negative returns right now.